### ISSUE TITLE
Make toolbar fixed in place if a11y services are enabled

### ISSUE THREAD

**Issue by Mugurell** [mozilla-mobile/fenix#18027]
Following https://github.com/mozilla-mobile/fenix/issues/17676:

> - If a user has accessibility services enabled and has not yet set the toolbar position to be at the bottom (although that is the default) or has set the toolbar position to be at the top:
    - the toolbar should be placed at _top_ in a fixed position (this is the current behavior)
> - If a user has accessibility services enabled and __has set the toolbar position to be at the bottom__
    - the toolbar should be placed at the _bottom_ in a fixed position (this is a difference to the current behavior)
> - In both cases the toolbar would be fixed in place ignoring the dynamic behavior set in app settings. (currently the a11y fixed top toolbar ignores the dynamic behavior set in app settings)

&nbsp;
[Eitan confirmed](https://github.com/mozilla-mobile/fenix/issues/17676#issuecomment-780054854) setting the bottom toolbar as fixed would be ok. 
*When users specifically chosen the toolbar to be shown at the bottom.
> I think this behavior makes sense. The one issue I can think of is that two not great things can happen when the toolbar is at the bottom with TalkBack enabled:
> 1. Either the UI layout will visually show the toolbar at the bottom, but talkback swipe order will first go through the toolbar before web content. This would be inconsistent with visual layout. Or..
> 2. The swipe order will correctly put content before the toolbar. The issue then would be that users who don't use explore-by-touch will need to swipe a variable and potentially large amount of times to reach the toolbar depending on what content is displayed.
>
> Honestly, I am fine with either outcome since that would be something the user explicitly opted in for in the settings.

**Comment by Mugurell** [comment]
Starting to work on this following the spike we had in the sprint.

**Comment by sflorean** [comment]
Verified as fixed on 87.0.0-beta.1 and latest Nightly build.
